scripts/create-component.js

```diff
@@ -0,0 +1,182 @@
+'use strict';
+
+process.on('exit', () => {
+  console.log();
+});
+
+const inquirer = require('inquirer');
+const path = require('path');
+const fs = require('fs');
+const fileSave = require('file-save');
+const uppercamelcase = require('uppercamelcase');
+const glob = require('glob');
+
+const FOLDER_NAME = 'Which folder';
+const COMPONENT_NAME = 'Component name (Such as Table)';
```

Comment:
Great! I just think it should get component atomic type (folder) and name from CLI arguments ([process.argv](https://nodejs.org/docs/latest/api/process.html#process_process_argv)), agree?
If you prefer I can take care of it later...
Thanks!

Reply:
Both are good to me. 
My only concern is that manual input is easy to be a typo so that I provide a  list. This is what I think about.

Reply:
I understood, I still think it would be easier with args, but it's not a problem at all, it's already very good this way. :)

Reply:
Thank for your understanding. I'll refactor it later and add it to the doc.

Reply:
@leomp12   Already refactor and add doc. Please check it again.

Reply:
Great!
I noticed you're up checking the atomic type, very good :)
I left only one last suggestion.